src/main/java/uk/gov/hmcts/reform/em/orchestrator/exampleservice/ExampleBundlePopulator.java

```diff
@@ -23,6 +23,9 @@ public ExampleBundlePopulator(ObjectMapper objectMapper) {
     public JsonNode populateNewBundle(JsonNode caseData) {
         CcdBundleDTO ccdBundleDTO = new CcdBundleDTO();
         ccdBundleDTO.setTitle("New Bundle");
+        ccdBundleDTO.setFileName("a-stitched-document.pdf");
```

Comment:
Do we want to assign a default, or leave it blank? If we give a default, I think it should be something that will help services realise that they should make it service-specific, like "exampleservice-bundle.pdf"